.github/workflows/makefile.yml

```diff
@@ -15,31 +19,59 @@ jobs:
     - name: shellcheck
       run: make -C build shellcheck
 
-  build:
+  build_containers:
     needs: shellcheck
     runs-on: ubuntu-latest
     strategy:
       matrix:
         os: [ubuntu2004, fedora33]
+        platform: [linux/amd64]
 
     steps:
-    - uses: actions/checkout@v2
+    - name: Checkout repository
+      uses: actions/checkout@v2
 
-    - name: Install dependencies
-      run: |
-        sudo echo 'debconf debconf/frontend select Noninteractive' | sudo debconf-set-selections
-        sudo apt-get -y install apt-utils
+    - name: Set up QEMU        
+      uses: docker/setup-qemu-action@v1
 
-    - name: Install Docker CE
-      run: |
-        sudo apt-get install ca-certificates curl gnupg lsb-release
-        curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
-        echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
-        sudo apt-get update
-        sudo apt-get install docker-ce docker-ce-cli containerd.io
+    - name: Set up Docker Buildx        
+      uses: docker/setup-buildx-action@v1
+
+#    - name: Login to DockerHub
+#      uses: docker/login-action@v1 
+#      with:
+#        username: ${{ secrets.DOCKERHUB_USERNAME }}
+#        password: ${{ secrets.DOCKERHUB_TOKEN }}
 
-    - name: Container Build
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1 
+      with:
+        registry: ghcr.io
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Install IPDK cli
       run: |
         cd build
         ./ipdk install ${{ matrix.os }}
-        ipdk build --no-cache
+ 
+    - name: Container build
+      run: |
+        ipdk build --no-cache --platform ${{ matrix.platform }}
+        ipdk tag ipdk-io/ipdk:${{ matrix.os }}-latest 
```

Comment:
I think we should tag this with the commit SHA has as well.

Reply:
Agree

Reply:
Implemented... The images are tagged with several tags depending on github actions event. One of the default tags us :sha-<short sha>